mne/io/tests/test_fieldtrip.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: UTF-8 -*-
+# Copyright (c) 2018, Thomas Hartmann & Dirk Gütlin
+# Authors: Thomas Hartmann <thomas.hartmann@th-ht.de>
+#          Dirk Gütlin <dirk.guetlin@stud.sbg.ac.at>
+#
+# License: BSD (3-clause)
+
+import mne
+import numpy as np
+import os.path
+from mne.datasets import testing
+
+
+@testing.requires_testing_data
```

Comment:
you need to add a decorator saying that this test requires hdf5. See how we do it in other test files for different libraries which are soft dependencies.